src/api/hardware/index.js

```diff
@@ -16,17 +16,13 @@
 
 import { ErgoDox } from "@api/hardware-ez-ergodox";
 import { Atreus2 } from "@api/hardware-keyboardio-atreus2";
-import {
-  Model01,
-  Model100,
-  Model100Bootloader,
-} from "@api/hardware-keyboardio-model01";
+import { Model01, Model100 } from "@api/hardware-keyboardio-model01";
 import { Splitography } from "@api/hardware-softhruf-splitography";
 import { Atreus } from "@api/hardware-technomancy-atreus";
 
 const Hardware = {
   serial: [Model01, Model100, Atreus2, ErgoDox, Atreus, Splitography],
-  nonSerial: [Model100Bootloader, ErgoDox, Atreus, Splitography],
+  nonSerial: [Model100, ErgoDox, Atreus, Splitography],
```

Comment:
Is the Model100 really both `serial` and `nonSerial` - That's confusing to me. Can you add some comments explaining what the two different arrays do?